packages/api/src/web5.ts

```diff
@@ -136,6 +141,12 @@ export type Web5ConnectOptions = {
    * See {@link TechPreviewOptions} for available options.
    */
   techPreview?: TechPreviewOptions;
```

Comment:
Sanity: should this be removed also?

Reply:
@thehenrytsai, @csuwildcat wanted us to keep it there for now so that existing code doesn't break.